tutorials/audio/audio_buses.rst

```diff
@@ -6,34 +6,23 @@ Audio buses
 Introduction
 ------------
 
-Beginning with Godot 3.0, the audio engine has been rewritten from scratch.
-The aim now is to present an interface much friendlier to sound design
-professionals. To achieve this, the audio engine contains a virtual rack
-where unlimited audio buses can be created and, on each of it, unlimited
-amount of effect processors can be added (or more like, as long as your
-CPU holds up!)
+Godot's audio processing code has been written with games in mind, with the aim of achieving an optimal balance between performance and sound quality.
```

Comment:
There were some audio changes in master, but AFAIK this page does not include them. Those changes are from a6ef7af39, and while the commit log is not very descriptive, it looks like a language/clarity copyedit.